synapse/lib/stormlib/ipv6.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import ipaddress
+
+
+import synapse.exc as s_exc
+
+import synapse.lib.stormtypes as s_stormtypes
+
+logger = logging.getLogger(__name__)
+
+@s_stormtypes.registry.registerLib
+class LibIpv6(s_stormtypes.Lib):
+    '''
+    A Storm Library for providing ipv6 helpers.
+    '''
+    _storm_locals = (
+        {'name': 'expand',
+         'desc': '''
```

Comment:
Any particular reason this is a tripple-quote / multi-line string?

Reply:
added note / example